benchexec/test_benchexec_integration.py

```diff
@@ -27,11 +27,15 @@
 import tempfile
 import unittest
 import zipfile
+
+from xml.etree import ElementTree
+
 sys.dont_write_bytecode = True # prevent creation of .pyc files
 
 here = os.path.dirname(__file__)
 base_dir = os.path.join(here, '..')
 bin_dir = os.path.join(base_dir, 'bin')
+tmp_dir = os.path.join(here, 'test_integration')
```

Comment:
All paths are relative, right? So there could be a copy of the directory structure re-created in the temp directory, such that all paths stay valid relative to each other?